torch/_tensor_docs.py

```diff
@@ -362,15 +362,13 @@ def add_docstr_all(method, docstr):
 
 Example::
 
-    >>> a = torch.randn(4, 2).byte() % 2
+    >>> a = torch.rand(1, 2).bool()
```

Comment:
these examples seem bogus -- you aren't really ever going to get False, right?

How about leaving these as randn and using a comparison operator (like `< 0`) instead of converting directly to bool?